src/Invitation/InvitationForm.js

```diff
@@ -105,12 +128,13 @@ export default function InvitationForm() {
             onChange={previewGallery}
             multiple
           />
+          <div>💡Pro Tip: A vertical photo will look best.</div>
           <div className="d-flex justify-content-center mt-2">
             <PlumButton onClick={addPicToGallery}>Add more photos</PlumButton>
           </div>
         </Form.Group>
         <div className="d-flex justify-content-end">
-          <Link to="/rsvp-preview" state={{ from: galleryURL }}>
+          <Link to="/event/1/rsvp-preview">
```

Comment:
for final product tomorrow, try to use the useParams() id, lmk if you need help with it tmr morning